package.json

```
@@ -4,12 +4,12 @@
   "scripts": {
     "test:unit": "jest",
     "test:lint": "eslint .",
-    "test:types": "tsc",
+    "test:types": "tsc --build packages/*",
     "pretest": "npm run build:bundle",
     "test": "npm run test:unit && npm run test:lint && npm run test:types",
     "build:bundle": "rollup -c",
-    "prebuild:types": "rimraf \"packages/**/*.d.ts\"",
-    "build:types": "tsc -p tsconfig.declarations.json",
+    "prebuild:types": "rimraf \"packages/**/*.d.ts\" \"packages/**/*.tsbuildinfo\"",
```

Could probably simplify this:

```suggestion
    "prebuild:types": "rimraf \"packages/**/*.+(d.ts|.tsbuildinfo)\"",
```

I expect it would be a marginal optimization as well, assuming it's the difference of finding files in a single vs. two passes.

https://github.com/isaacs/node-glob#glob-primer